app/controllers/AddressController.scala

```diff
@@ -731,7 +732,7 @@ class AddressController @Inject()(
                              "closure_of_correspondence",
                              auditForClosingPostalAddress(closingAddress, personDetails.etag, "correspondence")))
                      _        <- clearCache() //This clears ENTIRE session cache, no way to target individual keys
```

Comment:
Is it worth adding a ticket to enable a more focused clearing of the cache?

Reply:
Yeah, can we talk to Andy about raising a ticket for this?